src/Components/ImagesTable/Target.tsx

```diff
@@ -0,0 +1,54 @@
+import {
+  useGetClonesQuery,
+  useGetComposeStatusQuery,
+} from '../../store/apiSlice';
+
+type TargetProps = {
+  composeId: string;
+};
+
+const targetOptions = {
+  aws: 'Amazon Web Services',
+  'aws.s3': 'Amazon Web Services',
+  azure: 'Microsoft Azure',
+  gcp: 'Google Cloud Platform',
+  vsphere: 'VMWare',
+  'guest-image': 'Virtualization - Guest image',
+  'image-installer': 'Bare metal - Installer',
+  'edge-commit': 'edge-commit',
+  'edge-installer': 'edge-installer',
+  ami: 'ami',
+  'rhel-edge-commit': 'rhel-edge-commit',
+  'rhel-edge-installer': 'rhel-edge-installer',
+  vhd: 'vhd',
```

Comment:
Same, we never use the upload type for vhd, always the image type.

Reply:
aws.s3 can be safely left out, but if we leave out vhd, TypeScript shows a warning `Property 'vhd' does not exist on type`. 

We are getting the types from the ImageTypes enum in `imageBuilderSchema.ts` which was auto-generated by openapi-typescript. The vhd type is a part of ImageTypes values in `image-builder` so we could either leave it here even thought we never use it or remove it from the api specification as well to silence the error. Or at least I think those would be the options